### ISSUE TITLE
Add reverse routing capabilities to templates.

### ISSUE THREAD

**Issue by raphaelbauer** [ninjaframework/ninja#68]
root.put("Router", new BeansWrapper().getStaticModels().get("controllers.routes"));

and 

```
<h2>${products ? size} Products</h2>

<ul>
<#list products as product>
    <li><a href="${Router.Application.product(product.id)}">${product.name}</a></li>
</#list>
</ul>
```

**Comment by thibaultmeyer** [comment]
This feature don't seem to be implemented in the version 4.0.5. It will be added in the next release ? or they are another way to use reverse routing in the template without manually inject Router in every controllers

**Comment by danielsawan** [comment]
This feature exist : ${reverseRoute("controllers.ApplicationController", "methodName")}

Check the manual : http://puu.sh/fAY9r/81e77f9959.png
